gtest_parallel.py

```diff
@@ -270,7 +270,7 @@ def __register_exit(self, task):
     self.times.record_test_time(task.test_binary, task.test_name,
                                 task.last_execution_time)
     if self.test_results:
-      self.test_results.log(task.test_name, task.runtime_ms,
+      self.test_results.log(task.test_name, task.runtime_ms / 1000,
```

Comment:
There's some code in here that assumes that this script works with Python2 and Python3 both, I'm not sure if that's true anymore, but could you make this / 1000.0 to avoid truncation just in case Python2 happens to still work? Other than that this is good to merge. Thank you!

Reply:
Good point, I already switched to full python3 mode.